tinygraph-align.c

```diff
@@ -0,0 +1,18 @@
+#define _POSIX_C_SOURCE 200809L
```

Comment:
Maybe we should make the aligned alloc functions user-definable; at least at compile time?